src/TurbulenceClosures/turbulence_closure_implementations/convective_adjustment_vertical_diffusivity.jl

```diff
@@ -32,17 +32,17 @@ of `ConvectiveAdjustmentVerticalDiffusivity`. The default is `Float64`.
 Keyword arguments
 =================
 
-* `convective_κz` : Vertical tracer diffusivity in regions with negative (unstable) buoyancy gradients. Either
-                    a single number, function, array, field, or tuple of diffusivities for each tracer.
+* `convective_κz`: Vertical tracer diffusivity in regions with negative (unstable) buoyancy gradients. Either
+                   a single number, function, array, field, or tuple of diffusivities for each tracer.
 
-* `background_κz` : Vertical tracer diffusivity in regions with zero or positive (stable) buoyancy gradients.
+* `background_κz`: Vertical tracer diffusivity in regions with zero or positive (stable) buoyancy gradients.
```

Comment:
This may not be the place to discuss this, but is "background" the best word to use in this model? Maybe "stable" would be more descriptive?

Reply:
`stable` and `unstable` are more literal names for these quantities ("convective" and "background" refer to their physical interpretation).

Reply:
I think "convective" is a pretty good name. Intuitive and descriptive. I only raised the issue because imo "background" is kinda general and can mean many things. But I'm okay with you think there's no better name.